dialogflow/cloud-client/src/main/java/com/example/dialogflow/ContextManagement.java

```diff
@@ -17,41 +17,38 @@
 package com.example.dialogflow;
 
 // Imports the Google Cloud client library
+
 import com.google.cloud.dialogflow.v2.Context;
 import com.google.cloud.dialogflow.v2.ContextName;
 import com.google.cloud.dialogflow.v2.ContextsClient;
 import com.google.cloud.dialogflow.v2.SessionName;
+import com.google.common.collect.Lists;
 import com.google.protobuf.Value;
 
+import java.util.List;
 import java.util.Map.Entry;
-import net.sourceforge.argparse4j.ArgumentParsers;
-import net.sourceforge.argparse4j.inf.ArgumentParser;
-import net.sourceforge.argparse4j.inf.ArgumentParserException;
-import net.sourceforge.argparse4j.inf.MutuallyExclusiveGroup;
-import net.sourceforge.argparse4j.inf.Namespace;
-import net.sourceforge.argparse4j.inf.Subparser;
-import net.sourceforge.argparse4j.inf.Subparsers;
-
 
 /**
  * DialogFlow API Context sample.
  */
 public class ContextManagement {
-
   // [START dialogflow_list_contexts]
+
   /**
    * Lists contexts
+   *
    * @param sessionId Identifier of the DetectIntent session.
    * @param projectId Project/Agent Id.
+   * @return List of Contexts found.
    */
-  public static void listContexts(String sessionId, String projectId) throws Exception {
+  public static List<Context> listContexts(String sessionId, String projectId) throws Exception {
+    List<Context> contexts = Lists.newArrayList();
```

Comment:
What's your take on the return statement being inside vs outside the tags?

Reply:
I'd say leave the return outside.

Reply:
I think that sounds reasonable.

Reply:
From the perspective of having these regions be copy-pastable by developers, though, I would think the function tag and return statement would be useful so that they could just plop the function into their code directly. They would basically have to write a function wrapper around the code anyway regardless

Reply:
I see what you are saying, but I think the presumption is that they can copy paste the code into an existing code and get it working. Even if they copy the function as a whole they still have to create a class around it, so I don't think including the function signature really make's a significant impact.

Reply:
hey every keystroke counts :)

Reply:
I feel we should make sure this gets decided in the new rubric. (To have a solid answer going forward)